clients/python/Makefile

```diff
@@ -14,12 +13,13 @@ install:
 clean:
 	rm -rf src/mr_openapi
 
-.PHONY: build-mr
-build-mr:
-	cd ../../ && IMG_REGISTRY=${IMG_REGISTRY} IMG_VERSION=${IMG_VERSION} make image/build
+.PHONY: deploy-latest-mr
+deploy-latest-mr:
+	cd ../../ && IMG_VERSION=${IMG_VERSION} make image/build && LOCAL=1 ./scripts/deploy_on_kind.sh
+	kubectl port-forward -n kubeflow services/model-registry-service 8080:8080 &
```

Comment:
should we kill this after running the tests?

Reply:
I think a broader limitation I see atm, is that you cannot re-issue `make test-e2e` on the terminal after a first time: say some of the pytest failed, we amend something, and we want to try again.

I believe it would be awesome if scripts/deploy_on_kind.sh could handle those cases, ie:
- is there already the cluster? Don't try to create a kind cluster.
- is there already a namespace for the model registry? Don't return as-error, maybe leverage apply like in `kubectl create namespace "$MR_NAMESPACE" --dry-run=client -o yaml | kubectl apply -f -` ?
- is there already a port forward to 8080? I do not try to open a new one

wdyt?